packages/markdown-editor/src/MarkdownEditor/plugins/FloatingLinkEditorPlugin/FloatingLinkEditorPlugin.tsx

```diff
@@ -38,6 +39,8 @@ const FloatingLinkEditor = ({ editor, isLink, setIsLink, anchorElem }: FloatingL
   const [linkUrl, setLinkUrl] = useState('');
   const [isEditMode, setEditMode] = useState(false);
   const [lastSelection, setLastSelection] = useState<RangeSelection | GridSelection | NodeSelection | null>(null);
+  const isEditorFocused = useEditorFocus();
+  const shouldRenderFloatingLink = isLink && (isEditMode || isEditorFocused);
```

Comment:
nit: is it necessary to check for `isEditMode`?

Reply:
We need to check the `isEditMode`, otherwise when the user clicks the link buble to edit the href it would hide the input field because the editor loses the focus